sequencer/src/main/java/net/consensys/linea/sequencer/txselection/selectors/MaxBundleGasPerBlockTransactionSelector.java

```diff
@@ -60,15 +49,67 @@ public TransactionSelectionResult evaluateTransactionPostProcessing(
       final TransactionEvaluationContext txContext,
       final TransactionProcessingResult transactionProcessingResult) {
 
-    // short circuit if we are not a PendingBundleTx
-    if (!(txContext.getPendingTransaction() instanceof LineaSendBundle.PendingBundleTx)) {
-      return SELECTED;
-    }
```

Comment:
style-wise, having short circuit exits immediately by the condition make the code more readable IMO.  Not blocking, since the code has the same net result, just a readability preference.

Reply:
I changed it because I thought it was the only way to use the new version of `instanceof`, but actually it work also the other way, so I reintroduced the short circuit